### DISCUSSION TITLE
Remove all T3 mines

### DISCUSSION THREAD

**Discussion by Kami-Interface** [Team-QUIET/QUIET-Community-Edition#135] [Ideas]
The mines (from BrewLan) are never used in our games, but there may be a use case for the T1 mines (and possibly the T2) if we give them cloak with no E cost. However, by the time players have the economy to be building something like the T3 mine there's going to be omnis on the field. I don't see how the T3 mine would ever see real usage

**Comment by PurpleMossClump** [comment]
I've only ever found a single practical use for Nuke Mines, and it was in a game on Wraith Advance. When placed underwater, for intercepting football teams and other large masses of enemy units. It worked purely because the seabed was deep enough that spy plane omni couldn't reach it. 
That's such a narrow use-case that I don't see a good reason to keep them around other than because we can.
